neurom/view/view.py

```diff
@@ -72,12 +72,9 @@ def _scale_linewidth_with_axis(ax, linewidth):
         linewidth (float):  linewidth in data coordinates
     Returns (float): rescaled linewidth
     """
-    try:
+    if sys.version_info.major == 3:
         # this is done after first add_collection, but we need it here for latest matplotlib
```

Comment:
I think the `latest matplotlib` comment will go out date too fast, so some description of what is going on, and why it's needed, IMO

Reply:
just added more details. I was thinking of puting an issue on matplotlib, as I think it should be fixable from matplotlib side. I'll try to make a toy example that fails to send it to them and see what they say.

Reply:
I wonder if there is a better choice than `LineCollection` w/ `linewidth`, such that the units are scalled correctly, and we don't have to do any of this.  For instance `Rectange` scales correctly?

I haven't looked at it thoroughly, though.

Reply:
Yes, it seems rectangle will work: https://matplotlib.org/devdocs/api/_as_gen/matplotlib.patches.Rectangle.html
I can give it a try if you want, but I'm not sure I'll be able to do that correctly

Reply:
> Yes, it seems rectangle will work

Work as in correctly scale?  That's what I don't know, and don't have time to test.  If you're think it will, and you're willing to give it a shot, that would be appreciated.

Reply:
I'll give it a try and update this PR if it works better, I agree it will be cleaner to get rid of this rescale thing, I don't like it either